docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -1,63 +1,59 @@
 = Interact with the Unmodified {SpringBoot} Booster
 
-Once you have the {SpringBoot} booster deployed, you will have two applications running: `hello-service` and `name-service`. The `name-service` exposes a REST endpoint which returns a name as a String. The `hello-service` exposes a REST endpoint that returns a hello message as a String. When you call the REST endpoint in `hello-service`, the `hello-service` application makes a call to `name-service` to obtain a name String to use in the hello message String. 
+Once you have the {SpringBoot} booster deployed, you will have a service called  `{app-name}` running that exposes a two REST endpoints: `/api/greeting` which returns a name as a String and `/api/killme`, which forces the service to become unresponsive as means to simulate a failure.
 
-. Use `curl` to execute a `GET` request against the `hello-service`. You can also use a browser to do this.
+In the following we demonstrate the basic interactions on the command line that can be used verify the service availability and simulate a failure that causes the OpenShift self-healing capabilities to be trigger on the service. Alternatively you can use the service web interface to perform these steps (See #4).
+
+. Use `curl` to execute a `GET` request against the `{app-name}` service. You can also use a browser to do this.
 +
-[source,options="nowrap"]
+[source,options="nowrap",subs="attributes"]
 ----
-$ curl http://hello-service.192.168.99.100.nip.io/api/greeting
-Hello World!
+$ curl http://{app-name}-myproject.192.168.64.3.nip.io/api/greeting
+{"content":"Hello, World!"}
 ----
 
-. Scale down the number of `name-service` pods, which are used by `hello-service`, to 0.
+. To simulate a service internal failure and to trigger the OpenShift selfhealing capabilities, invoke the `/api/killme` endpoint and verify the availability of the `/api/greeting` endpoint shortly after that. It should return a HTTP status 503:
```

Comment:
You want your steps to be a clear action thats easily readable. Ideally, this means starting each step with an obvious verb. Use additional text within the step for more details. Also, you need to make your steps consistent with their endings (colons or periods). If you are doing sentences, each step should end in a period.

>  . Invoke the `/api/killme` endpoint and verify the availability of the `/api/greeting` endpoint shortly after that. 
> \+
> Invoking the `/api/killme` endpoint simulates an internal service failure and triggers the OpenShift self-healing capabilities. When invoking `/api/greeting` after simulating the failure, the service should return a HTTP status 503.